src/graph/DropSpaceExecutor.cpp

```diff
@@ -40,6 +40,8 @@ void DropSpaceExecutor::execute() {
         if (*spaceName_ == ectx()->rctx()->session()->spaceName()) {
             ectx()->rctx()->session()->setSpace("", -1);
         }
+        ectx()->addWarningMsg("Data will be deleted completely after restarting the services");
```

Comment:
BTW.  What's the message meaning?...

Reply:
when the user executes `drop space`, it will give the warning message to the user that the data dir will delete after service restart